eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -50,27 +53,26 @@ stages:
             environment: github
 
             pool:
-              vmImage: vs2017-win2016
+              vmImage: windows-2019
 
             strategy:
               runOnce:
                 deploy:
                   steps:
-                    - checkout: none
-                    - template: ../../tools/clone-buildtools/clone-buildtools.yml
+                    - checkout: azure-sdk-tools
                     - pwsh: |
                         Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
                         New-Item -Type Directory -Name ${{artifact.safeName}} -Path $(Pipeline.Workspace)
                         Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.name}}.[0-9]*.[0-9]*.[0-9]* $(Pipeline.Workspace)/${{artifact.safeName}}
                         Get-ChildItem $(Pipeline.Workspace)/${{artifact.safeName}}
                       displayName: Stage artifacts
                       timeoutInMinutes: 5
-                    - pwsh: |
-                        $(Pipeline.Workspace)/azure-sdk-build-tools/scripts/create-tags-and-git-release.ps1 -artifactLocation $(Pipeline.Workspace)/${{artifact.safeName}} -packageRepository Nuget -releaseSha $(Build.SourceVersion) -repoId $(Build.Repository.Name)
-                      displayName: 'Verify Package Tags and Create Git Releases'
-                      timeoutInMinutes: 5
-                      env:
-                        GH_TOKEN: $(azuresdk-github-pat)
+                    - template: eng/pipelines/templates/scripts/create-tags-and-git-release.yml@azure-sdk-tools
+                      parameters:
+                        ArtifactLocation: $(Pipeline.Workspace)/${{artifact.safeName}}
+                        PackageRepository: Nuget
+                        ReleaseSha: $(Build.SourceVersion)
+                        RepoId: $(Build.Repository.Name)
```

Comment:
Based on a recent issue we hit I think you may need to change this to be `Azure/azure-sdk-for-net` because `$(Build.Repository.Name)` will end up being set to azure-sdk-tools.

Reply:
Right!